card/lib/card.rb

```diff
@@ -4,20 +4,10 @@
 
 class Card < ActiveRecord::Base
   require_dependency 'card/active_record_ext'
-  require_dependency 'card/codename'
   require_dependency 'card/query'
-  require_dependency 'card/set_pattern'
-  require_dependency 'card/set'
-  require_dependency 'card/format'
   require_dependency 'card/exceptions'
-  require_dependency 'card/auth'
   require_dependency 'card/log'
-  require_dependency 'card/loader'
-  require_dependency 'card/content'
   require_dependency 'card/action'
-  require_dependency 'card/act'
-  require_dependency 'card/change'
-  require_dependency 'card/reference'
```

Comment:
I think I'm voting for the explicit route.  Autoloading is for rapid development, but we need a higher premium on reliability.

I would only want NOT to require the history stuff if we really want card to stand without the history stuff. That would be a bit of a leap.  I haven't really pondered that.